### DISCUSSION TITLE
12/05 반재영 블로그 2회차

### DISCUSSION THREAD

**Discussion by baaanjy** [Orm-camp/blog-feedback#46] [동글동글 3기 피드백]
https://velog.io/@baaanjy/웹-접근성-찍먹-경험

**Comment by mh0223** [comment]
aria-haspopup, aria-expanded, aria-label에 대해 꼼꼼하게 설명해주시고 직접 적용해본 경험이 드러나서 좋았습니다!

**Comment by kimjigyun** [comment]
안 보이는 곳까지 고려하여서 공부하신게 멋있고 잘 정리해주셔서 잘 읽고 갑니닷 !

**Comment by ghost** [comment]
적용 전, 후 사진이 있어서 이해하기 더 쉬운거 같습니다~
